.github/workflows/rust.yml

```diff
@@ -53,6 +54,10 @@ jobs:
       run: cargo build --no-default-features --features linux-${{ matrix.linkage }}-${{ matrix.library }} --verbose
     - name: Run tests
       run: cargo test --no-default-features --features linux-${{ matrix.linkage }}-${{ matrix.library }} --verbose
+    - name: Verify package
+      run: |
+        rm -f ./etc/libusb/README.md
```

Comment:
Its not entirely clear to me why you have to rm the README here. Can you maybe explain?

Reply:
Looks like a symbolic link to the actual `README` file in the libusb repo. Instead of simply renaming the file they have chosen this unorthodox approach.

It suddenly appears during one of the preceding build steps. Should either be fixed upstream or in this repository.

Passing `--allow-dirty` instead as a wildcard allowance workaround could be dangerous, not recommended.

Reply:
Ok, maybe just add a short comment to the CI to make that clear. After that we can merge the PR.

Reply:
I just found out myself, apparently its just done in the CI step to install libusb as a dependency. I think the problem was, that the old CI images in github actions had incompatible / old versions. 

Would probably be better to build that stuff in some kind of temporary directory. Not sure if mktemp would work.

Reply:
Done.

Reply:
Fixed: https://github.com/ruabmbua/hidapi-rs/pull/148/commits/19d0d3dfd4f0d048d2172f5142fd7532c58f63b3

The `etc/libusb/` directory in my local file system was probably a leftover from a previous checkout. Nothing to do here.